src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -2390,10 +2390,17 @@ public override async Task<DaprMetadata> GetMetadataAsync(CancellationToken canc
             try
             {
                 var response = await client.GetMetadataAsync(new Autogenerated.GetMetadataRequest(), options);
-                return new DaprMetadata(response.Id,
-                                        response.ActorRuntime.ActiveActors.Select(c => new DaprActorMetadata(c.Type, c.Count)).ToList(),
-                                        response.ExtendedMetadata.ToDictionary(c => c.Key, c => c.Value),
-                                        response.RegisteredComponents.Select(c => new DaprComponentsMetadata(c.Name, c.Type, c.Version, c.Capabilities.ToArray())).ToList());
+                if (response is null)
+                    return null;
+
+                return new DaprMetadata(response.Id ?? "",
```

Comment:
I don't quite know the standard for this repo but I know a lot of developers prefer to use `string.Empty` over `""`. Is that a valid recommendation for the Dapr .NET SDK?

Reply:
To my knowledge, there isn't a convention in the SDK either way with regards to that. Pretty much the only one I'm familiar with is private properties named using camel case without an underscore prefix.

I suspect the convention used throughout the rest of Dapr.Client will be "" for empty strings. That might be a worthwhile convention to set as we gradually migrate to individual clients for each feature, but I don't know if it's worth enforcing at this juncture (especially as it'd mean there's some inconsistency for a while).

Reply:
Sounds good to me.

Reply:
As a third alternative, I've always preferred `String.Empty`. :-)